pro_tes/tasks/track_task_progress.py

```diff
@@ -2,36 +2,36 @@
 
 import logging
 from time import sleep
-from typing import (
-    Dict,
-)
+from typing import Dict
 
 from foca.database.register_mongodb import _create_mongo_client
 from foca.models.config import Config
 from flask import Flask
 from flask import current_app
 import tes
 
-from pro_tes.ga4gh.tes.models import TesState
+from pro_tes.ga4gh.tes.models import TesState, TesTask
 from pro_tes.utils.db import DbDocumentConnector
 from pro_tes.ga4gh.tes.states import States
 from pro_tes.celery_worker import celery
+from pro_tes.utils.models import TaskModelConverter
 
 logger = logging.getLogger(__name__)
 
 
+# pylint: disable-msg=too-many-locals
 @celery.task(
     name="tasks.track_run_progress",
     bind=True,
     ignore_result=True,
     track_started=True,
 )
 def task__track_task_progress(
-    self,  # pylint: disable=unused-argument
-    worker_id: str,
-    remote_host: str,
-    remote_base_path: str,
-    remote_task_id: str,
+        self,  # pylint: disable=unused-argument
+        worker_id: str,
+        remote_host: str,
+        remote_base_path: str,
+        remote_task_id: str,
```

Comment:
Did `black` auto-format that in this way? I'm pretty sure there should be only one level of indentation relative to the function `def` here...

Reply:
Yes, you are right, I used black it reformatted to 1 line of indentation. fixing in the next commit

Reply:
Great, thanks